hana/exporter.sls

```diff
@@ -9,10 +9,16 @@
 {% set config_file = '/etc/hanadb_exporter/{}.json'.format(daemon_instance) %}
 
 hanadb_exporter:
-  pkg.installed
+  pkg.installed:
+  - retry:
+      attempts: 3
+      interval: 15
 
 python3-PyHDB:
-  pkg.installed
+  pkg.installed:
+ - retry:
```

Comment:
Could you indent properly this `retry` entry

Reply:
yep thx.. doing.. :crying_cat_face: